modules/ROOT/pages/to-download-and-install-studio-wx.adoc

```diff
@@ -12,7 +12,7 @@ endif::[]
 . Open the extracted Anypoint Studio.
 +
 When Studio first runs, the embedded OpenJDK libraries might cause a Windows Security alert. +
-Ensure to allow the OpenJDK Platform library from the AdoptOpenJDK publisher through your Windows firewall.
+Ensure to allow the OpenJDK Platform library from the Adoptium Eclipse Temurin OpenJDK publisher through your Windows firewall.
```

Comment:
It seems that it shows as `Publisher` just `Temurin` (this is an image I found on Internet. I wasn't able to replicate it by myself):
<img width="531" alt="image" src="https://user-images.githubusercontent.com/1738654/171919953-d935293b-6aa9-4f79-882a-68112db38f04.png">

This was the previous one for AdoptOpenJDK:
![image](https://user-images.githubusercontent.com/1738654/171919993-fcc861d2-6250-4139-b955-38195e4c7a20.png)

I guess we can leave it `Adoptium Eclipse Temurin` but maybe remove the `OpenJDK` from it. I leave it to your criteria.

Reply:
Thanks for the SS! Much clearer now for me!